core/src/main/java/org/mule/runtime/core/el/DataWeaveExpressionLanguageAdaptor.java

```diff
@@ -53,6 +50,8 @@
   public static final String ATTRIBUTES = "attributes";
   public static final String ERROR = "error";
   public static final String VARIABLES = "variables";
+  public static final String PROPERTY = "property";
+  public static final String PARAM = "param";
```

Comment:
let's use parameters since we don't have any other binding without the complete name